tests/unit/test.config.js

```diff
@@ -70,12 +71,15 @@ describe('config', () => {
       const configObject = {
         sourceDir: '/configured/source/dir',
       };
-      const newArgv = applyConf({argv, configObject, defaultValues});
+      const newArgv = applyConf({argv,
+        configObject,
+        defaultValues,
+        optionTypes});
```

Comment:
This adds a lot of verbosity. The `argv`, `defaultValues` and `optionTypes` variables are not actually needed for this test (and many others) so you could make it less verbose by passing them with a spread operator:
````js
const params = makeArgv();
...
const newArgv = applyConf({...params, configObject});
````